app/src/Commands/Repo/DeleteCommand.php

```diff
@@ -41,7 +44,38 @@ public function execute(InputInterface $input, OutputInterface $output) : int
     {
         $name = $input->getArgument("name");
         $global = $input->getOption("global");
+        $all = $input->getOption("all");
 
+        if (is_null($name) && ! $all) {
+            throw new InvalidArgumentException("Not enough arguments (missing: \"name\" or \"all\")");
+        }
+
+        if ($all) {
+            $repos = $this->repo_manager->getLocalRepos();
```

Comment:
Maybe avoid loading local repos if $global is true anyways?

Reply:
Avoiding if/else has an higher priority here as calling this cheap file function.